keycloak-extensions/pom.xml

```diff
@@ -49,6 +50,24 @@
       <groupId>ch.qos.logback</groupId>
       <artifactId>logback-classic</artifactId>
     </dependency>
+	<dependency>
+	  <groupId>org.seleniumhq.selenium</groupId>
+	  <artifactId>selenium-java</artifactId>
+	  <version>3.141.59</version>
+	  <scope>test</scope>
```

Comment:
I recently moved the dependency versions out of the child projects and into the `<dependencyManagement>` section of the parent pom.  Given that the project consists of just two modules, its not too important, but maybe you can move this up to the parent pom as well just for consistency?

Reply:
Hi Lee - am not very proficient with mvn parent/child project associations; i tried moving the dependencies into parent pom ; but i my tests fail to resolve the code specific to the dependencies; tried out mvn install and update and did not help. hence have not made any changes here. Need your help here. thanks!